bears/general/FilenameBear.py

```diff
@@ -19,14 +19,22 @@ class FilenameBear(LocalBear):
                           'snake': to_snakecase,
                           'space': to_spacecase}
 
+    _language_naming_convention = {
+                                   '.java': 'pascal',
+                                   '.js': 'kebab',
+                                   '.py': 'snake',
+                                  }
+
     def run(self, filename, file,
-            file_naming_convention: str='snake',
+            file_naming_convention: str=None,
```

Comment:
@jayvdb wanted a `None` option by default so we can warn the user to explicitly set the `file_naming_convention`